change/@fluentui-react-swatch-picker-preview-6246310c-e802-4b75-a4dd-6aa211c022b9.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "fix(react-swatch-picker): api changes and fixes",
```

change files should not contain scope ( as their are already scoped )
```suggestion
  "comment": "fix: api changes and fixes",
```